src/Resources/contao/dca/tl_page.php

```diff
@@ -1433,11 +1433,9 @@ public function copyPageWithSubpages($row, $href, $label, $title, $icon, $attrib
 			return '';
 		}
 
-		$objSubpages = $this->Database->prepare("SELECT * FROM tl_page WHERE pid=?")
-									  ->limit(1)
-									  ->execute($row['id']);
+		$subPages = \PageModel::countByPid($row['id']);
```

Comment:
This has no effect. If you want to reuse the existing model, you have to use `findByPid()` here.